### ISSUE TITLE
Remove "update release" from menu or mark as deprecated

### ISSUE THREAD

**Issue by tobiasfielitz** [CISOfy/lynis#447]
> lynis

```
Usage: lynis command [options]
    [..]
    update
        update info                   : Show update details
        update release                : Update Lynis release
    [..]
```

> lynis update release

`Deprecated: this function is no longer available. Use a package (https://packages.cisofy.com), or deploy via a custom package or script.`

It is confusing, that the option is shown in the menu but is deprecated.

_Version            : 2.5.3_

**Comment by mboelen** [comment]
Sounds like a good improvement. Want to create a pull request for it?

**Comment by tobiasfielitz** [comment]
https://github.com/CISOfy/lynis/pull/448

**Comment by mboelen** [comment]
Closed with #448
